### DISCUSSION TITLE
Failed webhooks notifications

### DISCUSSION THREAD

**Discussion by adrianpunkt** [supabase/supabase#29652] [Questions]
Is there a way to be notified whenever a webhook fails? I can see the request and error being tracked in `net._http_response`, but there's no way to know when that happens. 

Also it's not clear if there's any retry mechanism in place, it seems if a webhook request gets a `4xx` response for example, it will not be retried, nor notified. 

Am I missing something? Anyone has any workaround? Thanks!

**Comment by begank** [comment]
I can see the request and error being tracked in net._http_response
 ---------
 hello,I Got the same problem, and How can you see the request and error being tracked in net._http_response
